pages/accountLists/[accountListId]/reports/expectedMonthlyTotal/[[...contactId]].page.tsx

```diff
@@ -14,89 +13,89 @@ import {
   MultiPageMenu,
   NavTypeEnum,
 } from 'src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu';
+import {
+  ContactPanelProvider,
+  useContactPanel,
+} from 'src/components/common/ContactPanelProvider/ContactPanelProvider';
 import { useAccountListId } from 'src/hooks/useAccountListId';
-import { useContactLinks } from 'src/hooks/useContactLinks';
 import useGetAppSettings from 'src/hooks/useGetAppSettings';
-import { getQueryParam } from 'src/utils/queryParam';
-import { ContactsWrapper } from '../../contacts/ContactsWrapper';
 
 const ExpectedMonthlyTotalReportPageWrapper = styled(Box)(({ theme }) => ({
   backgroundColor: theme.palette.common.white,
 }));
 
-const ExpectedMonthlyTotalReportPage = (): ReactElement => {
+const PageContent: React.FC = () => {
   const { t } = useTranslation();
   const accountListId = useAccountListId();
-  const { appName } = useGetAppSettings();
+  const { isOpen } = useContactPanel();
   const [isNavListOpen, setNavListOpen] = useState<boolean>(false);
   const [designationAccounts, setDesignationAccounts] = useState<string[]>([]);
 
-  const { query } = useRouter();
-  const selectedContactId = getQueryParam(query, 'contactId');
-  const { handleCloseContact } = useContactLinks({
-    url: `/accountLists/${accountListId}/reports/expectedMonthlyTotal/`,
-  });
-
   const handleNavListToggle = () => {
     setNavListOpen(!isNavListOpen);
   };
 
+  return accountListId ? (
+    <ExpectedMonthlyTotalReportPageWrapper>
+      <SidePanelsLayout
+        isScrollBox={false}
+        leftPanel={
+          <MultiPageMenu
+            isOpen={isNavListOpen}
+            selectedId="expectedMonthlyTotal"
+            onClose={handleNavListToggle}
+            designationAccounts={designationAccounts}
+            setDesignationAccounts={setDesignationAccounts}
+            navType={NavTypeEnum.Reports}
+          />
+        }
+        leftOpen={isNavListOpen}
+        leftWidth="290px"
+        mainContent={
+          <ExpectedMonthlyTotalReport
+            accountListId={accountListId}
+            designationAccounts={designationAccounts}
+            isNavListOpen={isNavListOpen}
+            onNavListToggle={handleNavListToggle}
+            title={t('Expected Monthly Total')}
+          />
+        }
+        rightPanel={isOpen ? <DynamicContactsRightPanel /> : undefined}
+        rightOpen={typeof isOpen !== 'undefined'}
```

Comment:
Could you write this instead?


```suggestion
        rightOpen={!isOpen}
```